polyfill.js

```diff
@@ -0,0 +1,137 @@
+'use strict'
+
+if (typeof Symbol !== 'function' || !Symbol.isConcatSpreadable) {
+  throw new Error('Symbol.isConcatSpreadable missing from the environment')
+}
+
+if (typeof Array.prototype.flatMap !== 'function') {
+  // eslint-disable-next-line no-extend-native
+  Object.defineProperty(Array.prototype, 'flatMap', {
+    enumerable: false,
```

Comment:
Hm, it's not written in the spec, but true, it's better that way. Not specifying `enumerable: false` should also let it work with es5-sham.